config/webpack.dev.js

```diff
@@ -1,58 +1,55 @@
-const merge = require( 'webpack-merge' );
-const WebpackAssetsManifest = require( 'webpack-assets-manifest' );
-const webpack = require( 'webpack' );
-const common = require( './webpack.common.js' );
-const miniExtract = require( 'mini-css-extract-plugin' );
-const assetsData = Object.create( null );
-const DependencyExtractionWebpackPlugin = require(
-	'@wordpress/dependency-extraction-webpack-plugin'
-);
+const merge = require('webpack-merge');
+const WebpackAssetsManifest = require('webpack-assets-manifest');
+const webpack = require('webpack');
+const common = require('./webpack.common.js');
+const miniExtract = require('mini-css-extract-plugin');
+const assetsData = Object.create(null);
+const DependencyExtractionWebpackPlugin = require('@wordpress/dependency-extraction-webpack-plugin');
 
-const { requestToExternal, requestToHandle } = require(
-	'./asset-dependency-maps'
-);
-const { assetsManifestOutputPath } = require( './paths' );
+const { requestToExternal, requestToHandle } = require('./asset-dependency-maps');
+const { assetsManifestOutputPath } = require('./paths');
 
 const pluginsConfigWithExternals = [
-	new webpack.ProvidePlugin( {
+	new webpack.ProvidePlugin({
 		React: 'react',
-	} ),
-	new DependencyExtractionWebpackPlugin( {
+	}),
+	new DependencyExtractionWebpackPlugin({
 		requestToExternal,
 		requestToHandle,
-	} ),
-	new WebpackAssetsManifest( {
+	}),
+	new WebpackAssetsManifest({
 		output: assetsManifestOutputPath,
 		assets: assetsData,
-	} ),
-	new miniExtract( {
+	}),
+	new miniExtract({
 		filename: '[name].[contenthash].dist.css',
-	} ),
+	}),
 ];
 
 const pluginsConfigWithoutExternals = [
-	new WebpackAssetsManifest( {
+	new WebpackAssetsManifest({
 		output: assetsManifestOutputPath,
 		assets: assetsData,
-	} ),
-	new webpack.ProvidePlugin( {
+	}),
+	new webpack.ProvidePlugin({
 		React: 'react',
-	} ),
-	new miniExtract( {
+	}),
+	new miniExtract({
 		filename: '[name].[contenthash].dist.css',
-	} ),
+	}),
 ];
 
-common.forEach( ( config, index ) => {
-	const plugins = config.entry[ 'eventespresso-vendor' ] ?
-		pluginsConfigWithoutExternals :
-		pluginsConfigWithExternals;
+common.forEach((config, index) => {
+	const plugins = config.entry['eventespresso-vendor'] ? pluginsConfigWithoutExternals : pluginsConfigWithExternals;
 
-	common[ index ] = merge( config, {
+	common[index] = merge(config, {
 		devtool: 'inline-source-map',
-		plugins,
 		mode: 'development',
-	} );
-} );
+		plugins,
+		stats: {
+			logging: 'verbose',
+		},
```

Comment:
This adds: 
![image](https://user-images.githubusercontent.com/1477453/71447252-e9a66400-2734-11ea-85b0-fa9d72ad0f4c.png)